src/components/Board.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <div v-if="gameStats.gameStart">
     <h2>{{ msg }}</h2>
-    
+    <div class="wrapper">
+      <div v-for="(row, index) in gameSquares" :key="index" class="row">
+        <p>{{ index }}</p>
+        <div v-for="square in row" :key="square.id" :class="square.classes">
```

Comment:
once inside of a single row, I create a nested loop - this time for each square inside of the row. I don't need to call the `index` here because in the data block, each `square` has a unique ID already. So I am just looping over `v-for="square in row"` and pass the `square.id` as the `:key` value. 

Just like `:key` is a shorthand for `v-bind:key`, `:class` is a shorthand for `v-bind:class`. Here I call `:class` which can take a variable from the loop instead of a static string. `square.classes` will pull the string value of the `"classes"` property from each square object in the `gameSquares.json`. In this case there are only two possible outputs; `class="square red-square"` or `class="square black-square"`